src/main/java/de/tum/cit/aet/artemis/assessment/dto/FeedbackAnalysisResponseDTO.java

```diff
@@ -0,0 +1,11 @@
+package de.tum.cit.aet.artemis.assessment.dto;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.cit.aet.artemis.core.dto.SearchResultPageDTO;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public record FeedbackAnalysisResponseDTO(SearchResultPageDTO<FeedbackDetailDTO> feedbackDetails, long totalItems, int totalAmountOfTasks, List<String> testCaseNames) {
```

Comment:
Why is one attribute an int and the other a long? Do you expect such big numbers for one attribute?

Reply:
the page i use for pagination from artemis returns a long value for this attribute. Therefore i did not cast it